Example/TZDatePicker/ViewController.swift

```diff
@@ -37,6 +37,7 @@ class ViewController: UIViewController {
         let datePicker = TZDatePicker(dateComponents: dateComponents)
         datePicker.onValueChanged = {
             dateComponents in
+            dateComponents
```

Comment:
First of all, thank you very much for writing the README for me!! Now for the review: why did you add this line? Please remove it.